setup.py

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env python3
-from setuptools import setup
+import os
+
+from setuptools import setup, find_packages
+
+BASEDIR = os.path.abspath(os.path.dirname(__file__))
+
+
+def required(requirements_file):
+    """ Read requirements file and remove comments and empty lines. """
+    with open(os.path.join(BASEDIR, requirements_file), 'r') as f:
+        requirements = f.read().splitlines()
+        if 'MYCROFT_LOOSE_REQUIREMENTS' in os.environ:
```

Comment:
Is this still used now that versioning is fixed in ovos-core?

Reply:
this flag is used by most mycroft and ovos packages, it has been requested for ease of packaging in distros.

no matter what version we pin theres always some bleeding edge distro with a different one pinned, i think it makes sense to keep using this as a global flag in the mycroft/ovos/neon ecosystem

Reply:
This will just break core packages though, right? If a newer and unsupported package is installed, modules pinning an older version will raise exceptions at runtime

Reply:
this is mainly for distros (ie, not using pip but coming from package manager), if the distros have a newer version of any dependency in their repos they cant package this. 

Core can break, thats why this is a flag.... but 99.9% of the time historically its just a pinned random package with no impact at all, and if someone set this flag they want to find the breakage for fixing it, thats the distros job! if a distro wants to have core but use a newer version of the dependencies let them, they are supposed to have tested it and ensure the package combinations works

Even if the newer versions are compatible and everything should work the plugin will refuse to load and claim version conflicts if installed without this flag